bagnowka/pipeline-spec.yaml

```diff
@@ -18,6 +18,43 @@ download:
       parameters:
         input-resource: bagnowka
         output-resource: dbs-docs
+    - run: add_metadata
+      parameters:
+        name: bagnowka-convert
```

Comment:
I think you can remove all the add_metadata steps - they were required in older versions of the framework, but now I think they are not needed